lib/puppet/functions/extlib/commit_date.rb

```diff
@@ -0,0 +1,25 @@
+Puppet::Functions.create_function(:'extlib::commit_date', Puppet::Functions::InternalFunction) do
```

Comment:
‘puppet::Functions::InternalFunction’?

Reply:
This is apparently necessary to get access to `scope_param`. I didn't find another way to get the current environment name in puppet 4 functions.

Reply:
ah, ok.  Perhaps a short comment mentioning that.

Could you also add some puppet-strings style docs and regenerate REFERENCE.md?

Reply:
Is there still a commitment for this module to support puppet4 if not this can probably get dropped now